### PR TITLE
Updated tests for Version 2015-04-11

### PR DESCRIPTION
**Pull request by pon**

### CODE REVIEW

**Comment by pon** [comment]
@elnaz

File: tests/test_job.py [hunk 1]
```diff
@@ -64,14 +64,14 @@ def test_create_job_inline(self):
                 'address_state': 'CA'
             },
             objects = {
-                'name': 'Object1',
-                'file': 'https://s3-us-west-2.amazonaws.com/lob-assets/test.pdf',
+                'description': 'Object1',
+                'file': 'https://lob.com/postcardfront.pdf',
```
Comment (leore) [inline tests/test_job.py]: use HTML

File: tests/test_letter.py [hunk 2]
```diff
@@ -27,7 +27,8 @@ def test_create_letter(self):
                 'address_state': 'MA'
             },
             to_address = self.addr.id,
-            file = 'https://lob.com/goblue.pdf'
+            file = 'https://lob.com/goblue.pdf',
```
Comment (leore) [inline tests/test_letter.py]: use HTML

File: tests/test_postcard.py [hunk 3]
```diff
@@ -25,8 +25,8 @@ def test_create_postcard(self):
         postcard = lob.Postcard.create(
             to_address = self.addr.id,
             from_address = self.addr.id,
-            front = 'https://s3-us-west-2.amazonaws.com/lob-assets/test.pdf',
-            back = 'https://s3-us-west-2.amazonaws.com/lob-assets/test.pdf'
+            front = 'https://lob.com/postcardfront.pdf',
```
Comment (leore) [inline tests/test_postcard.py]: use HTML for all tests

**Comment by elnaz** [comment]
LGTM, thanks!

**Comment by pon** [comment]
Updated

File: lob/resource.py [hunk 4]
```diff
@@ -109,6 +109,7 @@ class CreateableAPIResource(APIResource):
     @classmethod
     def create(cls, **params):
         requestor = api_requestor.APIRequestor()
+        print params
```
Comment (leore) [inline lob/resource.py]: delete

Comment (pon) [inline lob/resource.py]: Fixed

File: lob/api_requestor.py [hunk 5]
```diff
@@ -68,7 +68,11 @@ def request(self, method, url, params=None):
                     if isinstance(v, resource.LobObject):
                         data[k] = v.id
                     else:
-                        data[k] = v
+                        if isinstance(v, dict):
+                            for k2, v2 in v.iteritems():
```
Comment (leore) [inline lob/api_requestor.py]: was metadata not wokring before than?

Comment (pon) [inline lob/api_requestor.py]: Metadata was - this is a double nested object (in the case of jobs/objects)

Comment (leore) [inline lob/api_requestor.py]: got it

**Pull request merged**